src/components/major/Highlights.js

```diff
@@ -189,232 +194,305 @@ const styles = StyleSheet.create({
-        </Select>
-      }
-      {highlightGroupsToShow.length === 0 &&
-        <Text style={styles.none}>
-          {i18n("None.", "", "enhanced")}
-        </Text>
-      }
-      {highlightGroupsToShow.length !== 0 &&
-        <ScrollView
-          style={styles.scrollView}
-          contentContainerStyle={wideMode ? styles.scrollViewContentWideMode : styles.scrollViewContent}
-        >
-          {highlightGroupsToShow.map(({ spineIdRef, highlights }, idx) => (
-            <View key={idx}>
-              <View style={styles.spineLine} />
-              <View style={styles.spineTextContainer}>
-                <Text
-                  style={styles.spineText}
-                  numberOfLines={1}
-                >
-                  {spineLabelsByIdRef[spineIdRef]}
-                </Text>
-              </View>
-              {highlights.map(({ cfi, types, text, notes, sketch, instructorHighlightersWithoutNotes }) => (
-                <View style={styles.highlight} key={cfi}>
-                  <View style={styles.types}>
-                    {types.map(type => (
-                      <Text
-                        style={styles[type]}
-                        key={type}
-                      >
-                        {typeStrings[type]}
-                      </Text>
-                    ))}
-                    <View style={styles.buttons}>
-                      <Button
-                        style={styles.button}
-                        size="small"
-                        status="basic"
-                        accessoryLeft={ReadIcon}
-                        appearance="ghost"
-                        onPress={goRead}
-                        info={{
-                          spineIdRef,
-                          cfi,
-                        }}
-                      />
-                      {/* <Button
+          )
+        : ({ style }) => (
+            <Icon
+              name="book-open-variant"
+              pack="materialCommunity"
+              style={style}
+            />
+          ),
+      [],
+    );
+    // const DrawIcon = useCallback(({ style }) => <Icon name="draw" pack="materialCommunity" style={style} />, [])
+    // const ShareIcon = useCallback(style => <Icon name="share" style={style} />, [])
+
+    const showSketch = useCallback(
+      ({ info: { sketch } }) => setSketchToShow(sketch),
+      [],
+    );
+
+    const goRead = useCallback(
+      ({ info }) => {
+        const newRouterState = goTo(info);
+        setSelectedToolUid({
+          // unselects any tool
+          bookId,
+          getRouterState: () => newRouterState,
+          historyReplace,
+        });
+      },
+      [bookId],
+    );
+
+    // const goShare = useCallback(
+    //   () => {
+
+    //   },
+    //   [],
+    // )
+
+    return (
+      <View style={wideMode ? styles.containerWideMode : styles.container}>
+        {selectOptions.length > 1 && (
+          <>
+            {Platform.OS === 'web' ? (
```

Comment:
I don't think the `<>` and `</>` is necessary here. Can we simplify the syntax as below?
```suggestion
        {selectOptions.length > 1 && (
            Platform.OS === 'web' ? (
```

Reply:
no we need <> to add Platform.OS